src/viur_cli/conf.py

```diff
@@ -94,6 +95,15 @@ def migrate(self):
         if (pyodide_version := self["default"].get("pyodide")) and pyodide_version.startswith("v"):
             self["default"]["pyodide"] = pyodide_version[1:]  # remove v prefix
 
+        #Check if a Cli version is in the project.json
+        if "cli-version" not in self:
+            self["cli-version"] = cli_version
+            print_changelog_from_github('viur-framework', 'viur-cli')
+
+        if "cli-version" in self and self["cli-version"] != cli_version:
```

Comment:
The previous `if` verifies this already!
```suggestion
        if self["cli-version"] != cli_version:
```